File: VagasWEB/src/fspoa/vagas/Cidade.java
package fspoa.vagas;

public class Cidade {
	
	private int id;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		if(checkIdCurso(id)){
			this.id = id;			
		}
		else{
			this.id = 0;
		}
	}
	
	public boolean checkIdCurso(int id){
		if(id > 0){
			return true;
		}
		return false;
	}
}

File: VagasWEB/cidade.php
<?php 
	
	class Cidade{
		private $idCidade;

		public function getId(){
			return $this->idCidade;
		}

		public function setId($id){

			self::checkIdCurso($id);
			$this->idCidade = $id;
			
		}

		public function checkIdCurso($id)
		{
			if($id <= 0){
				throw new Exception("Codigo inválido", 1);
				
			}
			
		}
	}
?>